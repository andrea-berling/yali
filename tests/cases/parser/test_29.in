(40 - 88) >= -(34 / 47 + 30)
