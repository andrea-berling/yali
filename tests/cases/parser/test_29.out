(>= (group (- 40.0 88.0)) (- (group (+ (/ 34.0 47.0) 30.0))))
